app/src/main/java/org/cru/godtools/ui/tooldetails/ToolDetailsFragment.kt

```diff
@@ -121,6 +123,16 @@ class ToolDetailsFragment() : BasePlatformFragment<ToolDetailsFragmentBinding>(R
             }
         }
     }
+
+    fun openTraining(tool: Tool?, primaryTranslation: Translation?) {
```

Comment:
hmm, I'm thinking passing in the manifest you want to open would make more sense than to use tool and primaryTranslation

Reply:
That would mean instead of doing a `hasTraining` variable on data binding, just put the `manifest` in data binding and evaluate if there are tips directly in data binding. You will also need to import `BuildConfig` and test the `DEBUG` constant to enable training for debug builds only